<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RegionalAreaSelect</class>
 <widget class="QDialog" name="RegionalAreaSelect">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>432</width>
    <height>768</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>疫情地图管理系统</string>
  </property>
  <widget class="QPushButton" name="exitButton">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>141</width>
     <height>54</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/GUI/UI_btn_cancel_round.png);</string>
   </property>
  </widget>
  <widget class="QLabel" name="placeName">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>20</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="UI_display">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>432</width>
     <height>768</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;img src=&quot;:/GUI/UI_display.png&quot;/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="nameLabel">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>90</y>
     <width>311</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>思源宋体 CN Heavy</family>
     <pointsize>36</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>默认</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="plainText">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>160</y>
     <width>331</width>
     <height>581</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>思源黑体 CN Regular</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="toolTipDuration">
    <number>-1</number>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(27, 27, 27);
color: rgb(255, 255, 255);</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
   <property name="plainText">
    <string/>
   </property>
   <property name="centerOnScroll">
    <bool>false</bool>
   </property>
  </widget>
  <zorder>placeName</zorder>
  <zorder>UI_display</zorder>
  <zorder>exitButton</zorder>
  <zorder>nameLabel</zorder>
  <zorder>plainText</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
